### PR TITLE
React app

### PR DESCRIPTION
**Pull request by eharley19**
Initial set-up of frontend

### CODE REVIEW

**Review by datdo — COMMENTED** [review]
LGTM

File: frontend/13f-app/src/reportWebVitals.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/reportWebVitals.js:13]: I'm gonna say we can probably go without this for now?
```suggestion
```

File: frontend/13f-app/src/components/MainSection.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.main-container {
+    background: url('/mnt/c/Users/eharl/documents/repos/filing13F_searcher/frontend/13f-app/src/images/img-1.jpg') center center/cover no-repeat;
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/MainSection.css:2]: This is going to break when the app is deployed. See if you can replace this with a path relative to the application itself.

File: frontend/13f-app/src/components/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+.nav-links-mobile {
+  display: none;
+}
+
+.menu-icon {
+  display: none;
+}
+
+@media screen and (max-width: 960px) {
```
Comment (TheNorthEestern) [inline frontend/13f-app/src/components/Navbar.css:78]: The nav becomes unusable/disappears once the device's screen becomes small enough.

**Comment by TheNorthEestern** [comment]
@eharley19 Let's get those changes in and get this merged.

**Comment by TheNorthEestern** [comment]
@eharley19 Close this out as we'll now be using the work you've done in #8

**Pull request merged**